src/app/donation.service.ts

```diff
@@ -230,14 +231,20 @@ export class DonationService {
       donationCouplets.findIndex(donationItem => donationItem.donation.donationId === donation.donationId),
       1,
     );
+
     this.storage.set(this.storageKey, donationCouplets);
+    this.sessionStorage.set(this.storageKey, donationCouplets);
   }
 
+  /**
+   * @todo This funtion can be deleted 1 day after this version is deployed - when we store only in session storage there is no need
+   * to remove anything.
+   */
```

Comment:
```suggestion
```

That isn't really true, we should keep deleting stuff indefinitely, since we can't rely on the browser to ever automatically remove what we put in local storage in the past.